EIPS/eip-7742.md

```diff
@@ -44,37 +44,57 @@ and deployment of changes to the blob count easier.
 
 #### Maximum blobs per block
 
-The blob maximum is verified in the CL node and the EL inherits this verification during the consistency check of the
-versioned hashes corresponding to each blob as specified by the Engine API. Because of this, the strict check specified
-by EIP-4844 is unnecessary.
+The max blob count is now provided and controlled by the CL during block production which the ELs must use to bound the maximum blobs it can pack in a block. 
```

Comment:
im fine to change the text but Im not sure why the original was edited?

Reply:
will restore, went through several edits in previous versions